### PR TITLE
Artifacts merges with OCI Image and Distribution Specs

### PR DESCRIPTION
**Pull request by SteveLasker**
Updates OCI Artifacts to reference Image and Distribution specs

### CODE REVIEW

**Comment by SteveLasker** [comment]
Adds a community-focused view, which can be merged with #68

File: README.md [hunk 1]
```diff
@@ -1,82 +1,11 @@
-# OCI Artifacts
+# OCI Artifacts Merges with Image and Distribution Specs
```
Comment (sabre1041) [inline README.md:1]: Recommend the title of the repository remain the same. Even with Artifacts becoming part of the Image and Distribution specification, is there a need to retain any of the past scope that is currently described within this README which is being replaced by the content below?

Depending on the decision, additional modifications to the content to follow

**Comment by SteveLasker** [comment]
Added feedback from @sabre1041, and merged in baseline content from @mikebrow

**Comment by SteveLasker** [comment]
Superceeded by #70

**Pull request closed without merging**